ideas/README.md

```diff
@@ -12,7 +26,7 @@ Or from one of the following sources:
 * [sleepyfox London Code Dojo](https://github.com/sleepyfox?utf8=%E2%9C%93&tab=repositories&q=dojo)
 * [Code Kata](http://codekata.com/)
 
```

Comment:
Maybe it would be helpful to mention some small'n'quick katas you can do on your own, at home, for funzies and get your code tested automatically by a system:

* [Code Wars](https://www.codewars.com/)
* [Project Euler](https://projecteuler.net/) (more algorithmic)
* [SPOJ](https://www.spoj.com/) (even more algorithimic)

Reply:
I feel like the point of code katas is the journey you take, not the correctness (although correctness has importance). 

I don't like code wars et al because they don't focus on the journey